### PR TITLE
Raise 'input' event when typing into a content editable element (closes #436)

### LINKED ISSUE #436: Input event is not raised during typing in a content editable element
(T336834)

### PR DESCRIPTION
**Pull request by AlexanderMoskovkin**
/cc @helen-dikareva @AndreyBelym

### CODE REVIEW

File: src/client/runner/automation/playback/type/type-char.js [hunk 1]
```diff
@@ -83,6 +85,13 @@ function _typeCharToContentEditable (element, text) {
     var startNode        = currentSelection.startPos.node;
     var endNode          = currentSelection.endPos.node;
 
+    // NOTE: some browsers raises 'input' event after the element
+    // content changes, but in the others we should do it manually.
```
Comment (AlexanderMoskovkin) [inline src/client/runner/automation/playback/type/type-char.js]: @VasilyStrelyaev please, check this comment
`element content changes` => `element content is changed`

**Comment by helen-dikareva** [comment]
lgtm

File: src/client/runner/automation/playback/type/type-char.js [hunk 2]
```diff
@@ -83,6 +85,23 @@ function _typeCharToContentEditable (element, text) {
     var startNode        = currentSelection.startPos.node;
     var endNode          = currentSelection.endPos.node;
 
+    // NOTE: some browsers raises 'input' event after the element
+    // content is changed, but in the others we should do it manually.
```
Comment (VasilyStrelyaev) [inline src/client/runner/automation/playback/type/type-char.js]:
```
    // NOTE: some browsers raise the 'input' event after the element
    // content is changed, but in others we should do it manually.
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by AlexanderMoskovkin** [comment]
FPR
/cc @VasilyStrelyaev

**Comment by AndreyBelym** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
thanks

**Pull request merged**